packages/babel-polyfill/src/index.js

```diff
@@ -1,7 +1,13 @@
+import "core-js/shim";
+import "regenerator-runtime/runtime";
+
 if (global._babelPolyfill) {
```

Comment:
Should we do 

```
if (global._babelPolyfill && typeof console !== "undefined" && console.warn) {
```
so that this doesn't throw on platforms that don't have `console`?